app/shared-components/ForgotPassword/ForgotPassword.jsx

```diff
@@ -60,45 +62,51 @@ class ForgotPassword extends React.Component {
 			this.props.actions.resetPassword(email)
 				.then((success) => {
 					this.setState({ loading: false });
-					if (success && locale === 'hub') {
-						this.props.history.push('/log-in');
+					if (success && isInHub) {
+						this.navigateToLogIn();
 
 						this.props.actions.setToast({
 							toastMessage: t('banner_check_your_email_title'),
 							toastClass: 'success',
 						});
-					} else if (success && locale === 'panel') {
+					} else if (success && isInPanel) {
 						this.props.history.push('/login');
 					}
 				});
 		});
 	}
 
+	navigateToLogIn = () => {
+		this.props.history.push('/log-in');
+	}
+
 	/**
 	 * Render Forgot Password panel.
 	 * @return {ReactComponent}   ReactComponent instance
 	 */
 	render() {
 		const { email, loading, emailError } = this.state;
 		const { locale } = this.props;
+		const isInPanel = locale === 'panel';
+		const isInHub = locale === 'hub';
```

Comment:
Why are we using "locale" as the variable name for which view we're in, when we already have locales for translations? I know this wasn't added in this PR, just wanted to point out that it's a rather confusing choice semantically.